fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -987,6 +988,9 @@ public void setEnableParallelMerge(boolean enableParallelMerge) {
     @VarAttr(name = ENABLE_MATERIALIZED_VIEW_REWRITE)
     private boolean enableMaterializedViewRewrite = true;
 
+    @VarAttr(name = ENABLE_SYNC_MATERIALIZED_VIEW_REWRITE)
+    private boolean enableSyncMaterializedViewRewrite = true;
```

Comment:
not a good name

Reply:
do you have a good idea? this config is only used to control whether to rewrite sync mv, so can be used in the test.

Reply:
ok, I misunderstood.